README.md

```diff
@@ -54,10 +54,13 @@ If you are using the GDS development virtual machine then the application will b
 
 ## Running the test suite
 
-Before you can run the test suite you'll need the [govuk-content-schemas]
-repository locally. See
-[`lib/govuk_content_schema_examples.rb`][content_schema_examples] for more
-details.
+Before you can run the test suite you'll need the [govuk-content-schemas] repository locally. See [`lib/govuk_content_schema_examples.rb`][content_schema_examples] for more details.
+
+The StandardJS JavaScript linter needs to be installed separately before running the default `rake` task. To make sure that it's been installed run:
+
+```sh
+$ npm install
```

Comment:
What about using `yarn` here? :D

Reply:
Good point! This version of Rails _should_ support Yarn out of the box. I'll test it and change it.

Reply:
That's awesome, I didn't know that :)

Reply:
<img width="307" alt="Screenshot showing yarn installing dependencies with four emojis" src="https://user-images.githubusercontent.com/1732331/57715913-adc00200-766f-11e9-920e-cc3598139b77.png">

Well, `yarn` apparently has emoji - so that's also a win too 👍